### ISSUE TITLE
Can't upgrade from python36u

### ISSUE THREAD

**Issue by kai4785** [iusrepo/python36u#10]
Since the package name has been changed to python36, I have systems that are not able to upgrade away from the existing python36u package.

What is the expected behavior or work around for this scenario?

I note that the philosophy page still says that the 'u' suffix has value, so I'm confused. Any additional information with respect to this would be helpful.

**Comment by carlwgeorge** [comment]
This change and it's justification is described in detail here:

https://github.com/iusrepo/packaging/issues/7

What problem are you currently having?  There isn't enough detail in this report for me to even guess what is wrong.  RHEL or CentOS?  6 or 7?  Have you checked for exclusions?  What command are you running?  What exact error do you see?  I tested the upgrade path several times, so if there is some unaccounted for variable I would love to know what I missed.

Regarding the website, the information is just out of date.  We are working on a complete site overhaul which is described here:

https://github.com/iusrepo/ius.io/issues/40

**Comment by kai4785** [comment]
Ok, that's great information. I wasn't able to find that information via google or browsing the website. After I filed that bug, and after seeing the comments about python3, I realized that the systems I'm having problems with are related to the python3 package from RHEL 7.7. My must have added a phantom '6' just to make my life miserable. I have rpm packages that are explicitly listing 'Requires: python36u', but getting 'Marking python3-libs-3.6.8-10.el7.x86_64 as install for <mypackage>' because we link against libpython, so I get 'Requires: libpython3.6m.so.1.0()(64bit)' in my package, and python3 is a newer version than python36u. So this is an issue I'll get to fix on my side.

Given your upgrade path, I have something to work with. Thanks!

**Comment by carlwgeorge** [comment]
@kai4785 If you haven't been able to sort this out yet, I posted remediation steps here:

https://github.com/iusrepo/packaging/issues/7#issuecomment-545038676

**Comment by kai4785** [comment]
Thanks for offering help. I was able to figure things out. It boiled down to our internal mirror being out of date when we generate disk images, and then spinning up systems from those images, and expecting them to install updates from online.

A few months ago, we punted on dealing with the ius repo folder hierarchy and gpg key file name changes, and then promptly forgot about it. That resulted in installing an older ius-release package on systems that expected to update online, which would have required updating the ius-release package before seeing the properly obsoleted python36u package available as an update, causing the conflicts with RHEL 7.7's python3.

tl;dr My bad. : )
